testkit/tests/currency/cryptocurrency.rs

```diff
@@ -144,7 +145,7 @@ impl Transaction for TxTransfer {
 
     /// Retrieve two wallets to apply the transfer. Check the sender's
     /// balance and apply changes to the balances of the wallets.
-    fn execute(&self, view: &mut Fork) {
+    fn execute(&self, view: &mut Fork) -> ExecutionResult {
```

Comment:
Similarly, transfers is a good place to showcase multiple errors:
- Unknown sender
- Unknown receiver
- Insufficient funds
- Arithmetic overflow for the receiver's balance (not tested now, btw)